### PR TITLE
drop all desktop related files and references

### PR DESCRIPTION
**Pull request by jakubgs**
Since desktop is now officially developed in [nim-status-client](https://github.com/status-im/nim-status-client) we can finally get rid of all of the desktop crust from this repo.
My favorite part of this is that we finally get rid of the symlinking of files from `mobile/js_files`.

Thanks for help from @flexsurfer with cleaning up the Clojure side of things.

And once this is merged I'll have to update Snyk settings so it can find the files.

Also, I snuck in a reduction in built timeouts to 20 minutes.

### CODE REVIEW

**Comment by jakubgs** [comment]
Closing in favor of https://github.com/status-im/status-react/pull/10907 which fixes the crashes.

**Comment by jakubgs** [comment]
Wrong issue, meant https://github.com/status-im/status-react/pull/10904.

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/pairing/views.cljs [hunk 1]
```diff
@@ -74,13 +69,10 @@
...
 
-(defn your-device [{:keys [installation-id name device-type]}]
+(defn your-device [{:keys [installation-id name]}]
   [react/view {:style styles/installation-item}
    [react/view {:style (styles/pairing-button true)}
-    [icons/icon  (if (= "desktop"
-                        device-type)
-                   :main-icons/desktop
-                   :main-icons/mobile)
+    [icons/icon :main-icons/mobile
```
Comment (cammellos) [inline src/status_im/ui/screens/pairing/views.cljs]: Not so fast :) 
this one it just means you are paired with a desktop machine, I think we should keep it, although desktop has no device pairing yet.

File: src/status_im/ui/screens/pairing/views.cljs [hunk 2]
```diff
@@ -94,17 +86,13 @@
...
                           enabled?
                           installation-id]}]
   [react/touchable-highlight
    {:accessibility-label :installation-item}
    [react/view {:style styles/installation-item}
     [react/view {:style (styles/pairing-button enabled?)}
-     [icons/icon (if (= "desktop"
-                        device-type)
-                   :main-icons/desktop
-                   :main-icons/mobile)
```
Comment (cammellos) [inline src/status_im/ui/screens/pairing/views.cljs]: same here

**Review by cammellos — CHANGES_REQUESTED** [review]
All good, the only thing is the installations/pairing code, I don't think we should remove the icons for desktop, while is true that you can't sync currently, as soon as you can we should show the right icon.

**Comment by jakubgs** [comment]
@flexsurfer could you please address the comments from @cammellos about clojure changes?

**Comment by flexsurfer** [comment]
@jakubgs done

File: src/status_im/ui/screens/pairing/views.cljs (hunk 1 above)
Comment (jakubgs) [reply src/status_im/ui/screens/pairing/views.cljs]: Addressed by Andrey.

File: src/status_im/ui/screens/pairing/views.cljs (hunk 2 above)
Comment (jakubgs) [reply src/status_im/ui/screens/pairing/views.cljs]: Addressed by Andrey.

**Review by cammellos — APPROVED** [review]

**Comment by Serhy** [comment]
I've checked latest build (https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200707-104549-1b55e1-pr-universal.apk) and iOS build is the same, - and there is a regression bug:
After re-login wallet is gone:
Screenshot 2020-07-07 at 14 53 50

**Comment by flexsurfer** [comment]
looking

**Comment by flexsurfer** [comment]
thanks @Serhy fixed

**Comment by jakubgs** [comment]
Nice, still waiting for a review of https://github.com/status-im/status-react-jenkins/pull/11 to merge it and tag it tho...

**Comment by jakubgs** [comment]
The only thing I changed was the branch name from `status-react-jenkins` into a tag for the same commit.
I don't see how could affect the e2e test results in the slightest, so I assume it's a fluke and will merge.

**Pull request merged**